test/test_parse_eu.py

```diff
@@ -0,0 +1,228 @@
+#
+# Copyright 2017 Mycroft AI Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from datetime import datetime
+import unittest
+
+from lingua_franca.parse import (normalize, extract_numbers, extract_number,
+                                 extract_datetime, isFractional_eu)
```

Comment:
Hey, the problem is that this function definition does not exist in the `lingua_franca.parse` namespace so it can't import `isFractional_eu` from this module. You can see that from the test output here: 
https://github.com/MycroftAI/lingua-franca/runs/7036546990?check_suite_focus=true

You would need to have:
```
from lingua_franca.parse import (normalize, extract_numbers, extract_number,
                                 extract_datetime)
from lingua_franca.lang.parse_eu import isFractional_eu
```

Or we'd need to register `is_fractional` as a top-level localizable function, but it's not really intended as a public function. At least I haven't seen it being used as such.

We should probably take the opportunity to fix up the casing of the function name too. It should really be `is_fractional_eu`. But I know this was copied from other languages.

Reply:
Done!

Also fixed some failing tests (which were not happening when the PR was made almost a year ago; I guess they were caused by some errors in the merging of the updates happened since).